src/code/z_bgcheck.c

```diff
@@ -1947,13 +1912,14 @@ s32 BgCheck_CheckWallImpl(CollisionContext* colCtx, u16 xpFlags, Vec3f* posResul
     dz = posNext->z - posPrev->z;
 
     // if there's movement on the xz plane, and argA flag is 0,
-    if (((dx != 0.0f) || (dz != 0.0f)) && !(argA & 1)) {
-        if ((checkHeight + dy) < 5.0f) {
+    if ((dx != 0.0f || dz != 0.0f) && !(argA & 1)) {
+        if (checkHeight + dy < 5.0f) {
             //! @bug checkHeight is not applied to posPrev/posNext
             result = BgCheck_CheckLineImpl(colCtx, xpFlags, COLPOLY_IGNORE_NONE, posPrev, posNext, &posIntersect, &poly,
                                            &bgId, actor, 1.0f, BGCHECK_CHECK_ALL & ~BGCHECK_CHECK_CEILING);
             if (result) {
-                ny = COLPOLY_GET_NORMAL(poly->normal.y);
+                // unit normal of polygon
+                f32 ny = COLPOLY_GET_NORMAL(poly->normal.y);
                 // if poly is floor, push result underneath the floor
```

Comment:
```suggestion
                f32 ny = COLPOLY_GET_NORMAL(poly->normal.y);

                // if poly is floor, push result underneath the floor
```
Spaces after declarations